src/storage/create.js

```diff
@@ -4,13 +4,19 @@ const { uploadDocument } = require("./documentService");
 
 const handleCreate = async event => {
   try {
-    const { document, ttl } = JSON.parse(event.body);
-    const receipt = await uploadDocument(document, Number(ttl));
+    const { document, id, ttl } = JSON.parse(event.body);
+    const receipt = await uploadDocument(document, id, Number(ttl));
```

Comment:
@rjchow just saw the test that this should work without ttl also. in this case if ttl wont passed the above code would through error.

Reply:
why?

Reply:
Number(ttl) this thorws error when ttl is undefined.

Reply:
i just get NaN which evaluates to false later on, how are you getting the error?

Reply:
Serverless: [400] {"statusCode":400,"body":"{\"error\":\"A value provided cannot be converted into a number\"}","headers":{"Access-Control-Allow-Origin":"*"}}


<img width="993" alt="Screenshot 2019-09-30 at 15 47 36" src="https://user-images.githubusercontent.com/12955874/65858908-a8065700-e399-11e9-8d8e-47929e117522.png">

Reply:
I am testing with postman ..and it throws error until I handle it.

Reply:
might be some weird interaction with NaN and validateTtl, you can decide how to handle it